drivers/tty/serdev/pibridge_trace.h

```diff
@@ -302,6 +502,13 @@ DECLARE_EVENT_CLASS(pibridge_kfifo_data_class,
 	)
 );
 
+/*
+ * pibridge_discard_data
+ *
+ * Info: the bytes contained in the receive fifo being discarded.
+ * Time: after access bytes have been discarded or a timeout occured while
+ *       waiting for the discard.
```

Comment:
Nit: s/access/excess/, s/occured/occurred/

But for clarity I'd add a half sentence to explain that the excess data is caused by a larger than expected packet. Also, "discard" as a noun seems wrong (means "Ausschuss").

Suggestion:

after the excess bytes of a larger than expected packet have been discarded or a timeout occurred while waiting for the data to be discarded.